File: 6-kyu/twin-prime/python/solution.py
def is_prime(n):
    if n < 2: return False
    prime = True
    for i in range(2, n//2 + 1):
        if n % i == 0:
            prime = False
            break
    return prime


def is_twinprime(n):
    return is_prime(n) and (is_prime(n-2) or is_prime(n+2)) 

File: 6-kyu/twin-prime/javascript/solution.js
const isPrime = (n) => {
  if (n <= 2) return false;

  let prime = true;
  for (let i = 2; i < Math.floor(n/2); i++){
      if (n % i == 0){
          prime = false;
          break;
      }
  }
  return prime;
}

const isTwinPrime = n => 
isPrime(n) && (isPrime(n - 2) || isPrime(n + 2));